x-pack/plugins/apm/public/components/app/diagnostics/summary_tab/index.tsx

```diff
@@ -39,3 +56,68 @@ export function DiagnosticsSummary() {
     </EuiFlexGroup>
   );
 }
+
+function CrossClusterSearchCallout() {
+  return (
+    <EuiCallOut title="Cross cluster search not supported" color="warning">
+      The APM index settings is targetting remote clusters. Please note that
+      this is not currently supported by the Diagnostics Tool and functionality
+      will therefore be limited.
+    </EuiCallOut>
+  );
+}
+
+function PrivilegesCallout({
+  diagnosticsBundle,
+}: {
+  diagnosticsBundle: DiagnosticsBundle;
+}) {
+  const missingClusterPrivileges = Object.entries(
+    diagnosticsBundle.diagnosticsPrivileges.cluster
+  )
+    .filter(([privilegeName, hasPrivilege]) => !hasPrivilege)
+    .map(([privilegeName]) => privilegeName);
+
+  const missingIndexPrivileges = Object.entries(
+    diagnosticsBundle.diagnosticsPrivileges.index
+  )
+    .filter(([indexName, privObject]) => !privObject.read)
+    .map(([indexName, privObject]) => indexName);
+
+  return (
+    <>
+      <EuiCallOut title="Insufficient access" color="warning">
```

Comment:
Thanks for adding the call out in the summary tab. 

The index templates and data streams tab shows a warning even when the user doesn't have access to retrieve this information which can lead to incorrect and misleading results.  

I would suggest either showing an empty message/response or hiding entirely the tab that the user doesn't have enough privilege. Not sure which is easier. What do you think?

<img width="1783" alt="image" src="https://github.com/elastic/kibana/assets/3369346/45d52cae-e962-4f39-a89a-c9dbb5c9f3c5">

Reply:
Fixed. Now the tabs are hidden if the user doesn't have all cluster privileges.